internal/nodes/avi_model_l4_translator.go

```diff
@@ -287,18 +296,12 @@ func PopulateServersForNPL(poolNode *AviPoolNode, ns string, serviceName string,
 		annotations = obj.([]lib.NPLAnnotation)
 		for _, a := range annotations {
 			var atype string
-			if utils.IsV4(a.NodeIP) {
-				if ipFamily != "V4" {
-					utils.AviLog.Infof("Skipping server %s, ipFamily is %s", a.NodeIP, ipFamily)
-					continue
-				}
+			if v4Family && utils.IsV4(a.NodeIP) {
 				atype = "V4"
-			} else {
-				if ipFamily != "V6" {
-					utils.AviLog.Infof("Skipping server %s, ipFamily is %s", a.NodeIP, ipFamily)
-					continue
-				}
+			} else if v6Family && utils.IsV6(a.NodeIP) {
 				atype = "V6"
+			} else {
+				continue
```

Comment:
Should we add warning message if len(poolMeta) is zero before line number 318, so that customer will get an idea that for specific IPFamily there are no pool server found. Should we do it for all variants of populateServer?

Reply:
added a warning log message

Reply:
This warning message will be printed for each annotation and each pool member. My intention was that when we coomplete all processing for pool before exiting out, currently we are printing log message `servers for port`. Before that message, if we print message for len(poolmeta) == 0, warning message saying we didn;t get any pool member for given ipfamily for given pod.

Reply:
@akshayhavile we already have something similar on L319

Reply:
have added new logs